src/components/Hero/Abilities.jsx

```diff
@@ -34,8 +34,8 @@ const renderAbilities = abilities => abilities.map(ability => (
 ));
 
 const Abilities = ({ hero }) => {
-  const filterAbilities = toFilterAbs => toFilterAbs.filter(ability => (ability !== 'generic_hidden'));
-
+  const filterAbilities = (toFilterAbs) =>
+    toFilterAbs.filter((ability) => ability !== 'generic_hidden' && abilities[ability].behavior !== 'Hidden');
```

Comment:
We can remove generic_hidden too?

Reply:
We can, but it's more efficient to leave it in since it's a quick check that doesn't require an object lookup in a large json. Happy to remove though, an object lookup shouldn't be too bad since it's a direct key substitute.